index.html

```diff
@@ -158,49 +166,127 @@ <h3 class="text-xl md:text-2xl font-semibold font-redhat text-gray-900">How does
+                            <p class="text-base md:text-lg">Our service monitors different metrics such as redeemable debt in front, market price, mempool, time of last adjustment to decide when, if and how to update the interest rate.</p>
                         </div>
                     </details>
+                </div>
+
+            <div class="divide-y divide-gray-200">
+                <details class="group py-4">
+                    <summary class="flex justify-between items-center cursor-pointer list-none">
+                        <h3 class="text-xl md:text-2xl font-semibold font-redhat text-gray-900">How much does it cost?</h3>
+                        <span class="transform group-open:rotate-45 transition-transform">
+                            <svg class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 4v16m8-8H4"/>
+                            </svg>
+                        </span>
+                    </summary>
+                    <div class="mt-4 text-gray-600">
+                        <p class="text-base md:text-lg">We charge a management fee of 0.25 % on your loan. We expect users will make a net profit from lower interest rates and gas savings.</p>
+                    </div>
+                </details>
+            </div>
 
+            <div class="divide-y divide-gray-200">
+                <details class="group py-4">
+                    <summary class="flex justify-between items-center cursor-pointer list-none">
+                        <h3 class="text-xl md:text-2xl font-semibold font-redhat text-gray-900">Is it safe to use?</h3>
+                        <span class="transform group-open:rotate-45 transition-transform">
+                            <svg class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 4v16m8-8H4"/>
+                            </svg>
+                        </span>
+                    </summary>
+                    <div class="mt-4 text-gray-600">
+                        <p class="text-base md:text-lg">The Liquity protocol enforces that all interest rate managers can only update the interest rate.
+                            They can *not* steal your money or manage any other aspect of your loan. The system enforces that all interest rate managers
+                            need to be registered with certain parameters. This includes the lower and upper bounds of the interest rate,
+                            the frequency of changes as well as the management fee. The management fee can be lowered afterwards but not increased. All
+                            other parameters can not be changed after registration so that users can not be tricked. All parameters can be verified on the blockchain.
```

Comment:
```suggestion
                            other parameters can not be changed after registration, so that users can not be tricked. All parameters can be verified on the blockchain.
```